### ISSUE TITLE
adding nuxt-security ^1.3.0 without any config leads to 500 (Netlify) and script errors (localhost) - expected?

### ISSUE THREAD

**Issue by denisenepraunig** [Baroshem/nuxt-security#419]
Hello, I am getting started with `nuxt-security`.

## TLDR;
I just installed the latest nuxt and nuxt-security, locally I get some warnings in the console but the site works, deployed to Netlify I get a 500.

❓ Without touching anything, I am wondering if this is the expected behaviour? 

Source code is here:
https://github.com/denisenepraunig/nuxt-security-test

Deployed to Netlify - 500
https://nuxt-security-test.netlify.app/

I've created a project from scratch with:
```console
npx nuxi@latest init nuxt-security-test
npx nuxi@latest module add security
```

which results in this `package.json` - `dependencies`:
```json
"dependencies": {
  "nuxt": "^3.11.2",
  "nuxt-security": "^1.3.0",
  "vue": "^3.4.21",
  "vue-router": "^4.3.0"
}
```

`nuxt.config.ts`:
```javascript
export default defineNuxtConfig({
  devtools: { enabled: true },
  modules: ["nuxt-security"]
})
```

### Localhost script errors but works
When I run it locally `npm run dev` I see this:

<img width="934" alt="Screenshot 2024-04-06 at 19 21 47" src="https://github.com/Baroshem/nuxt-security/assets/7646025/6cc57393-cf38-498c-9b9a-6272d0a354a2">

```console
Refused to load the script 'http://localhost:3002/_nuxt/@vite/client' because it violates the following Content Security Policy directive: "script-src 'self' https: 'unsafe-inline' 'strict-dynamic' 'nonce-{{nonce}}'". Note that 'strict-dynamic' is present, so host-based allowlisting is disabled. Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

localhost/:9 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https: 'unsafe-inline' 'strict-dynamic' 'nonce-{{nonce}}'". Either the 'unsafe-inline' keyword, a hash ('sha256-ywX1jMWBvLA4vnpd6j5yBtD3YAo2d1bvP29VM4VZ4so='), or a nonce ('nonce-...') is required to enable inline execution.

localhost/:19 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https: 'unsafe-inline' 'strict-dynamic' 'nonce-{{nonce}}'". Either the 'unsafe-inline' keyword, a hash ('sha256-ChValuzd0cyaDyQRGsHBMsFX2hzljF1GK3M80fFrnAk='), or a nonce ('nonce-...') is required to enable inline execution.
```

### Netlify 500
When I deploy to Netlify I get a 500

<img width="1063" alt="Screenshot 2024-04-06 at 19 31 09" src="https://github.com/Baroshem/nuxt-security/assets/7646025/503dfac0-fdb7-4a29-b7d4-d2958b54160c">

```console
nuxt-security-test.netlify.app/:1 Refused to execute inline script because it violates the following Content Security Policy directive: "script-src 'self' https: 'unsafe-inline' 'strict-dynamic' 'nonce-{{nonce}}'". Either the 'unsafe-inline' keyword, a hash ('sha256-gwEUKwLFu0zI6vy2K6p/qLT9Tc+uykTl+06eLad4k4I='), or a nonce ('nonce-...') is required to enable inline execution.
nuxt-security-test.netlify.app/:1 
 Failed to load resource: the server responded with a status of 500 ()
```

❓ Is this the expected behaviour?

**Comment by vejja** [comment]
Hi @denisenepraunig 
This is not normal. We are facing a dependency regression with 1.3.0, we will patch soon.
In the meantime can you please downgrade to  `"nuxt-security": "1.2.2"` in package.json ?

**Comment by vejja** [comment]
@Baroshem 
Related to Issue #415

**Comment by Baroshem** [comment]
Released patch 1.3.1 with a fix for that from @vejja.

Please check if it works now :)

**Comment by vejja** [comment]
I can confirm this scenario by @denisenepraunig now works correctly with patch 1.3.1

**Comment by Dino-Kupinic** [comment]
works!

**Comment by cesxhin** [comment]
<img width="701" alt="Screenshot 2024-04-23 alle 11 05 57" src="https://github.com/Baroshem/nuxt-security/assets/33133311/4309df27-446b-48f0-8350-68420faf0c11">

I'm used version nuxt-security: v1.3.2 but it happens when I want to specify contentSecurityPolicy inside.
Maybe regression?

```
security: {
        headers:{
            contentSecurityPolicy: {
                'upgrade-insecure-requests': false,
            },
        }
    }
```

**Comment by Baroshem** [comment]
@cesxhin I think this is a different issue. Could you please open a new idsue with all the details and preferably a reproduction repository?
